src/DateTimeSelect.php

```diff
@@ -15,12 +15,12 @@
  *     ...
  * }
  * @psalm-type InputArray = array{
- *     year: numeric,
- *     month: numeric,
- *     day: numeric,
- *     hour: numeric,
- *     minute: numeric,
- *     second: numeric
+ *     year: numeric-string,
+ *     month: numeric-string,
+ *     day: numeric-string,
+ *     hour: numeric-string,
+ *     minute: numeric-string,
+ *     second: numeric-string,
```

Comment:
Not convinced these should be `string` :think

Reply:
👍 - I've reverted and restricted the template also to `numeric`